### PR TITLE
Font variant

### PR DESCRIPTION
**Pull request by jacobp100**
Ground work for allowing `font-variant`s. Currently allows switching between `tabular-nums` and `proportional-nums`. I will need guidance on how to test this, and a few pointers on code style (new to Objective C, and had to make one or two hacks).

### CODE REVIEW

File: React/Base/RCTConvert.m [hunk 1]
```diff
@@ -649,6 +654,33 @@ + (UIFont *)UIFont:(UIFont *)font withFamily:(id)family
         fontDescriptor = [fontDescriptor fontDescriptorWithSymbolicTraits:symbolicTraits];
         font = [UIFont fontWithDescriptor:fontDescriptor size:fontSize];
       }
+
+      NSDictionary *RCTFontVariantMapping = @{
```
Comment (jacobp100) [inline React/Base/RCTConvert.m]: What’s the best way of tackling this in Objective C land?

**Comment by facebook-github-bot-1** [comment]
@jacobp100 updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@jacobp100 updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@jacobp100 updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@jacobp100 updated the pull request.

**Comment by jacobp100** [comment]
Messed up with git. Will open a new PR.

**Pull request closed without merging**